eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -12,7 +12,7 @@ stages:
     dependsOn: ${{parameters.DependsOn}}
     jobs:
       - deployment: SignPackage
-        environment: ${{parameters.Environment}}
+        environment: esrp
```

Comment:
I assume esrp doesn't require approval. If so do we need to have an environment set at all?